src/chip/src/components/Chip.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+import React from 'react';
+
+import { Chip as MuiChip, makeStyles } from '@material-ui/core';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/pro-light-svg-icons';
+import Colors from './colors';
+import type { ChipProps } from './types';
+
+const DeleteIcon = <FontAwesomeIcon fixedWidth icon={faTimes} />;
+
+const actionStates = (backgroundColor) => ({
+  '&:hover, &:focus': {
+    backgroundColor
+  },
+  '&:active': {
+    backgroundColor
+  },
+});
+
+const useStyles = (color) => {
+  const colors = Colors[color] || {};
+  return makeStyles({
+    root: {
+      ...colors
+    },
+    clickable: {
+      cursor: 'pointer',
+      ...actionStates(colors.clickable)
+    },
+    deletable: actionStates(colors.background),
+    deleteIcon: {
+      color: colors.color,
+      margin: '0 12px 0 -6px',
+    }
```

Comment:
I think there's an issue with the `deleteIcon` style. It seems MUI sets a `height` which makes the icon look bigger than it should be.

<img width="1101" alt="Screen Shot 2021-03-22 at 12 58 31 PM" src="https://user-images.githubusercontent.com/440299/112050631-4f3af900-8b0e-11eb-9dcb-42120099bced.png">

Reply:
Update: removing `fixedWidth` seems to fix this issue.